### PR TITLE
[docs] Change the incubator fields before and after Doris' Graduation

### PR DESCRIPTION
**Pull request by yuanyuan8983**
Changed the incregator field of Doris' compilation module, LDC toolchain compilation module and arm platform compilation module before and after Doris' Graduation

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/install/source-install/compilation.md [hunk 1]
```diff
@@ -35,34 +35,34 @@ This document focuses on how to code Doris through source code.
...
-| apache/incubator-doris:build-env-1.1 | [ff0dd0d](https://github.com/apache/incubator-doris/commit/ff0dd0d2daa588f18b6db56f947e813a56d8ec81) or later | 0.10.x or later |
-| apache/incubator-doris:build-env-1.2 | [4ef5a8c](https://github.com/apache/incubator-doris/commit/4ef5a8c8560351d7fff7ff8fd51c4c7a75e006a8) or later | 0.12.x - 0.14.0 |
-| apache/incubator-doris:build-env-1.3.1 | [ad67dd3](https://github.com/apache/incubator-doris/commit/ad67dd34a04c1ca960cff38e5b335b30fc7d559f) or later | 0.14.x |
-| apache/incubator-doris:build-env-for-0.15.0 | [a81f4da](https://github.com/apache/incubator-doris/commit/a81f4da4e461a54782a96433b746d07be89e6b54) or later | 0.15.0          |
-| apache/incubator-doris:build-env-latest | before [0efef1b](https://github.com/apache/incubator-doris/commit/0efef1b332300887ee0473f9df9bdd9d7297d824) | |
-| apache/incubator-doris:build-env-for-1.0.0| | 1.0.0 |
-| apache/incubator-doris:build-env-ldb-toolchain-latest | trunk | trunk |
+| apache/incubator-doris:build-env | before [ff0dd0d](https://github.com/apache/doris/commit/ff0dd0d2daa588f18b6db56f947e813a56d8ec81) | 0.8.x, 0.9.x |
```
Comment (yinzhijian) [inline docs/en/docs/install/source-install/compilation.md:54]: Why not change 'incubator-doris:build-env' to doris:build-env?

Comment (morningman) [reply docs/en/docs/install/source-install/compilation.md:54]: Some old image has not been moved to the new repo.
And these images are too old, may be removed later.

**Review by hf200012 — DISMISSED** [review]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**